src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -628,17 +654,28 @@ public String getMailAddressAttributeName() {
         return StringUtils.defaultString(mailAddressAttributeName, DescriptorImpl.DEFAULT_MAILADDRESS_ATTRIBUTE_NAME);
     }
 
+    /**
+     * Creates security components.
+     * @return Created {@link SecurityComponents}
+     * @throws Error Execution error
+     */
+    @Override @Nonnull
     public SecurityComponents createSecurityComponents() {
         Binding binding = new Binding();
         binding.setVariable("instance", this);
 
-        BeanBuilder builder = new BeanBuilder(Jenkins.getInstance().pluginManager.uberClassLoader);
+        final Jenkins jenkins = Jenkins.getInstance();
+        if (jenkins == null) {
+            throw new Error("Jenkins instance is not ready");
```

Comment:
"Urgh" on the existing `throw new Error(...)` - anyway - I'm not voting bug on this :-)